README.md

```diff
@@ -100,6 +100,30 @@ class ExponentialBackoffWorker
 end
 ```
 
+You can also customize the backoff function defining the `backoff_function` option, that can be any `call`able object (a lambda, a method, a class that responds to `call`, etc.)
+that will receive the current attempt count and should return in how many seconds the message will be retried. 
+
+```diff
+class ExponentialBackoffWorker
+  include Sneakers::Worker
+
+  from_queue "sneakers_handlers.my_queue",
+      ack: true,
+      exchange: "sneaker_handlers",
+      exchange_type: :topic,
+      routing_key: "sneakers_handlers.backoff_test",
+      handler: Sneakers::Handlers::ExponentialBackoffHandler,
++     backoff_function: ->(attemp_number) { attemp_number ** 3 },
+      max_retries: 50,
```

Comment:
The default is actually 5 for the `RetryHandler` and 25 for the `ExponentialBackoffHandler`. The example was just trying to show that you can customize that. Do you think we should use a different number?